composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "rakit/validation": "^1.2"
```

Comment:
Zou dit wel in de `require-dev` zetten en dan even nadenken hoe dit af te vangen.

(Je doet in je template `{% if app.debug %}` maar dat heeft niks te maken met wel of niet een `--dev` Composer install. Wellicht een `class_exixst()` check ergens?)

Reply:
Aan de andere kant... er staat nergens dat we de twig filter, die deze PR toevoegd, alleen in debug modus gebruikt mag worden...  

een `class_exists` check kan wel... maar is niet heel duidelijk (i.e. je voegt een stukje kennis toe, de filter bestaat alleen wanneer je in je eigen project ook een `composer require rakit/validation` uitvoert.  Als je dat niet doet, dan gaat het goed op je development machine, maar krijg je op productie een error... het is wel behoorlijk kak om er juist op productie achter te komen dat iets niet werkt...

Reply:
> wanneer je in je eigen project ook een composer require rakit/validation uitvoert

Nee dat zeker niet! Ik bedoel ik de `require-dev` zetten. Dus op development en testing heb je em dan wel (automagisch) omdat daar een composer install (default --dev) wordt gedaan. Op acceptatie en productie wordt een composer install --no-dev gedaan, dus wordt ie dan niet geïnstalleerd. Dus dat zou je dan moeten afvangen zodat je inderdaad geen errors krijgt. `class_exists` kan hierbij heel goed worden gebruikt (geeft geen errors).

Reply:
We praten langs elkaar, ik loop wel even langs :yum:

Reply:
Conclusie :point_up_2: was: niet in `require-dev` + detectie, want dat is niet transparant.